src/components/useFeedback.tsx

```diff
@@ -0,0 +1,37 @@
+import type {RefObject} from 'react';
+import {useEffect} from 'react';
+import * as Sentry from '@sentry/browser';
+import type { OverrideFeedbackConfiguration } from '@sentry/types';
```

Comment:
This isn't exported anymore (per https://github.com/getsentry/sentry-javascript/pull/12197)

So we'll need some kind of change.